xcube/api/gen/config.py

```diff
@@ -80,7 +79,7 @@ def get_config_dict(config_obj, open_function):
         except ValueError:
             output_size = None
         if output_size is None or len(output_size) != 2:
-            raise ValueError(f'invalid output_size {config_obj.output_size!r}')
+            raise ValueError(f'invalid output_size {config_obj.get("output_size")!r}')
```

Comment:
The value referred to in the error message is a not the object being tested in the `if` condition.